spec/index.html

```diff
@@ -68,7 +68,16 @@
     in terms of Namespaces in XML,
     the XML Information Set [[XML-INFOSET]]
     and XML Base [[XMLBASE]].</p>
-  <p>RDF 1.2 XML adds support for
+
+  <p>RDF 1.2 XML introduces <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a>
```

Comment:
The `RDF 1.2 XML` construction seems odd, given that the serialization is called `RDF/XML`, not `XML`.

I would expect either `RDF 1.2 XML` or, more likely, `RDF/XML 1.2`.

This might want its own issue.

Reply:
I see.

I suggest that (versionless, i.e., including all versions) `RDF/XML` be treated as an informal/short name for (versionless, i.e., including all versions) `RDF xxx XML Syntax`, and that (version specific) `RDF/XML xxx` be treated as an informal/short name for (version specific) `RDF xxx XML Syntax`.

I feel like without `Syntax`, `RDF xxx XML` (which does occur twice in this paragraph, and once circa line 1205) will be more confusing than not.